geofrontcli/cli.py

```diff
@@ -329,6 +329,8 @@ def ssh(args, alias=None):
         options = get_ssh_options(remote)
     except ValueError as e:
         ssh.error(str(e))
+    if args.jump_host:
+        options.extend(['-o', 'ProxyJump==%s' % (args.jump_host)])
```

Comment:
python3에서도 퍼센트로 문자열 포매팅이 되긴합니다만, `str.format`을 사용하며 포매팅 해주실 수 있을까요?

```suggestion
        options.extend(['-o', 'ProxyJump=={}'.format(args.jump_host)])
```